cinema/serializers.py

```diff
@@ -33,7 +33,16 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "image",
+            "duration",
+            "genres",
+            "actors",
+        )
+        read_only_fields = ("image",)
```

Comment:
Shouldn't `"id"` also be specified here?

Reply:
yees😄 
my fault
thank youuu)